src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+public class Deadline extends Task {
```

Comment:
I like how your code adheres well to the required coding standards. Well done! :)